vignettes/extras/BugAndPathwayAbundanceAllGutSamples_ar12.19.16.Rmd

```diff
@@ -52,11 +52,11 @@ for (i in 1:length(eset.list)){
     eset.list[[i]]$subjectID <- as.character(eset.list[[i]]$subjectID)
 }
 ```
-Keep leaf nodes only:
+
+Remove rows providing genus/species names:
 ```{r}
 for (i in seq_along(eset.list)){
-  eset.list[[i]] <- eset.list[[i]][grep("t__", rownames(eset.list[[i]]), invert=TRUE), ]
-  eset.list[[i]] <- eset.list[[i]][grep("s__|_unclassified\t", rownames(eset.list[[i]]), perl=TRUE), ]
+  eset.list[[i]] <- eset.list[[i]][!grepl("\\|", rownames(eset.list[[i]])), ]
```

Comment:
The "|" get clobbered to "." later, but can still use them here. Would take a little more effort to keep the pipes later, if it seems worthwhile.